editor/plugin_config_dialog.cpp

```diff
@@ -329,11 +292,9 @@ PluginConfigDialog::PluginConfigDialog() {
 	for (int i = 0; i < ScriptServer::get_language_count(); i++) {
 		ScriptLanguage *lang = ScriptServer::get_language(i);
 		script_option_edit->add_item(lang->get_name());
-#ifdef MODULE_GDSCRIPT_ENABLED
-		if (lang == GDScriptLanguage::get_singleton()) {
+		if (lang->get_name() == "GDScript") {
```

Comment:
This is unrelated to this PR, but I noticed that this method should likely be changed to return `StringName`.